src/quo/components/buttons/button/style.cljs

```diff
@@ -58,60 +58,61 @@
 
 (defn style-container
   [{:keys [size disabled? border-radius background-color border-color icon-only? icon-top
-           icon-left icon-right]}]
-  (merge {:height             size
-          :align-items        :center
-          :justify-content    :center
-          :flex-direction     (if icon-top :column :row)
-          :padding-horizontal (when-not (or icon-only? icon-left icon-right)
-                                (case size
-                                  56 (if border-color 10 11)
-                                  40 16
-                                  32 12
-                                  24 7
-                                  16))
-          :padding-left       (when-not (or icon-only? icon-left)
-                                (case size
-                                  56 nil
-                                  40 16
-                                  32 12
-                                  24 8
-                                  16))
-          :padding-right      (when-not (or icon-only? icon-right)
-                                (case size
-                                  56 nil
-                                  40 16
-                                  32 12
-                                  24 8
-                                  16))
-          :padding-top        (when-not (or icon-only? icon-left icon-right)
-                                (case size
-                                  56 0
-                                  40 (if border-color 8 9)
-                                  32 (if border-color 4 5)
-                                  24 0
-                                  (if border-color 8 9)))
-          :padding-bottom     (when-not (or icon-only? icon-left icon-right)
-                                (case size
-                                  56 0
-                                  40 9
-                                  32 5
-                                  24 0
-                                  9))
-          :overflow           :hidden
-          :background-color   (if disabled? (colors/alpha background-color 0.3) background-color)
-          :border-radius      (if border-radius
-                                border-radius
-                                (case size
-                                  56 12
-                                  40 12
-                                  32 10
-                                  24 8
-                                  12))
-          :border-color       border-color
-          :border-width       (when border-color 1)}
-         (when icon-only?
-           {:width size})
-         (when border-color
-           {:border-color border-color
-            :border-width 1})))
+           icon-left icon-right inner-style]}]
+  [{:height             size
```

Comment:
Likely not a big deal, but the original code returned a map (singular) and the name of the reflected that. With the new solution a vector is returned, but the name of the function is still in the singular, shouldn't it be pluralized, like `container-styles`? Still respecting this convention of pluralizing when sequences are returned can help prevent simple mistakes.